examples/armada-manifest-v1.yaml

```diff
@@ -0,0 +1,101 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-dependecy
+data:
+  name: helm-toolkit
+  release: helm-toolkit
+  namespace: null
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: helm-toolkit
+    reference: master
+  dependecies: []
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-mariadb
+data:
+  name: mariadb
+  release: mariadb
+  namespace: undercloud
+  timeout: 3600
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: mariadb
+    reference: master
+  dependecies:
+    - ucp-dependecy
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-memcache
+data:
+  name: memcache
```

Comment:
Should there also be some differentiation in the name/release?  Helm doesn't support duplicate release names as far as I'm aware, so would be nice to see this tagged as ucp-<service> or osh-<service>, or some other variants that distinguish between what's specific for the undercloud and what's specific for the control plane -- unless I'm missing the intent